### PR TITLE
lambda function tips for S3 triggers

### PR DESCRIPTION
**Pull request by DorrinPk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1245,6 +1245,7 @@ Lambda
 -	🔸Managing lots of Lambda functions is a workflow challenge, and tooling to manage Lambda deployments is still immature.
 -	🔸AWS’ official workflow around managing function [versioning and aliases](https://docs.aws.amazon.com/lambda/latest/dg/versioning-aliases.html) is painful.
 -	❗📜 Currently [as of October, 2016](https://github.com/open-guides/og-aws/pull/199/files/c99bddb4ee2437587f1e188d47be2bb1da01f81d#r83529126) Lambda functions can sometimes stop working for 2-3 minutes for failure recovery purposes according to a support ticket answer from Lambda development team. They are working to prevent this in the future.
+- 🔸 While adding/removing S3 buckets as triggers for lambda function, this error may occur : "There was an error creating the trigger: Configuration is ambiguously defined. Cannot have overlapping suffixes in two rules if the prefixes are overlapping for the same event type." In this case, remove the trigger manually by going to the Properties section of the S3 bucket, under Events, remove the lambda event.
```
Comment (ThanosBaskous) [inline README.md]: Micronits (no need to fix ahead of merge):
"may occur : " -> "may occur: " 
"lambda" -> "Lambda" (throughout)
'Properties' -> '"Properties"'
'Events' -> '"Events"'

**Comment by jlevy** [comment]
Looks good to me, aside from copy editing details. Thanks @DorrinPk !

**Comment by jlevy** [comment]
Over to you @ThanosBaskous

**Comment by ThanosBaskous** [comment]
@DorrinPk thanks again for putting this together - if you wouldn't mind resolving the conflicts when you get a chance, I can merge immediately after.

**Comment by DorrinPk** [comment]
@ThanosBaskous conflicts resolved.

**Comment by ThanosBaskous** [comment]
Thanks @DorrinPk!

**Pull request merged**